docs/cloud/pouta/persistent-volumes.md

```diff
@@ -22,6 +22,16 @@ is the size of the volume.
 
 ![Create persistent Volmume](../../img/create-volume-horizon.png)
 
+!!! Warning "No äkköset in name or description allowed"
+    This is a know bug in the volume system. Volume creation will fail
+    if its name or description contains any non ASCII characters.
```

Comment:
I think "non ASCII" is technically a compound adjective in this sentence and thus probably requires the hyphen. `non-ASCII`. Same holds for other occurrences in the rest of the PR

Reply:
Added, thanks.